packages/firebase_storage/firebase_storage/lib/src/firebase_storage.dart

```diff
@@ -149,6 +149,33 @@ class FirebaseStorage extends FirebasePluginPlatform {
     return _delegate.setMaxDownloadRetryTime(time.inMilliseconds);
   }
 
+  /// Changes this instance to point to a Storage emulator running locally.
+  ///
+  /// Set the [host] (ex: localhost) and [port] (ex: 5001) of the local emulator.
```

Comment:
standard storage emulator port 

```suggestion
  /// Set the [host] (ex: localhost) and [port] (ex: 9199) of the local emulator.
```